contracts/test/PoolLiquidityProvider.t.sol

```diff
@@ -184,16 +184,19 @@ contract PoolTest is TestUtil {
         // Mint 200 USDC into this LP account
         mintFundsAndDepositIntoPool(leo, pool, 20_000_000 * USD, depositAmt);
         mintFundsAndDepositIntoPool(liz, pool, 20_000_000 * USD, depositAmt);
-        
+
         assertEq(pool.depositDate(address(leo)), startDate);
         assertEq(pool.depositDate(address(liz)), startDate);
+        assertEq(pool.balanceOf(address(leo)), toWad(depositAmt));
+        assertEq(pool.balanceOf(address(liz)), toWad(depositAmt));
 
-        uint256 newDeposit  = constrictToRange(depositAmt, 50 * USD, depositAmt, true);  // Amount of FDT transferred
+        uint256 newDeposit = constrictToRange(depositAmt, 50 * USD, depositAmt, true);  // Amount of FDT transferred
 
-        hevm.warp(startDate + 30 days);
+        // Will fail because lockup period hasn't passed yet
+        assertTrue(!liz.try_transfer(address(pool), address(leo), toWad(newDeposit)));
 
-        assertEq(pool.balanceOf(address(leo)), toWad(depositAmt));
-        assertEq(pool.balanceOf(address(liz)), toWad(depositAmt));
+        // Warp to after lockup period
+        hevm.warp(startDate + pool.lockupPeriod());
```

Comment:
We should add another failure assert for `hevm.warp(startDate + pool.lockupPeriod() - 1);`